modules/freewheelSSPBidAdapter.js

```diff
@@ -47,11 +47,13 @@ function getPricing(xmlNode) {
   var princingData = {};
 
   var extensions = xmlNode.querySelectorAll('Extension');
-  extensions.forEach(function(node) {
+  // Nodelist.forEach is not supported in IE and Edge 
```

Comment:
`no-trailing-spaces` lint error on this line. The Atom eslint plugin is catching it, and it fails when built locally, but Travis build still isn't for some reason